tests/test_packages/test_skills/test_erc1155.py

```diff
@@ -30,6 +30,7 @@
 class TestERCSkillsEthereumLedger(AEATestCaseMany, UseOef):
     """Test that erc1155 skills work."""
 
+    @pytest.mark.flaky(reruns=5)  # cause possible network issues
```

Comment:
We should maximally rerun this 2 times. If it fails then it fails due to congestion most likely and reruns within close proximity will be impacted by a similar level of congestion.